packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -916,6 +925,18 @@ export const addEdge = async (id: string, edge: WorkflowEdge) => {
 	return response.data ?? null;
 };
 
+export const addOrUpdateAnnotation = async (id: string, annotation: WorkflowAnnotation) => {
+	console.log('>> workflowService.addOrUpdateAnnotation');
```

Comment:
Assuming this is intentional for a short time but figured I would point out console.log

Reply:
Noted - they will be here for maybe another week. 

But please do not fall into the trap of pointing out console.log or System.out.println, there are much more important things to pay attention on.